### ISSUE TITLE
Reshape layer determining dimensions of outputs

### ISSUE THREAD

**Issue by momenabdelkarim** [lutzroeder/netron#142]
I'm not sure if this is where I should be asking this question, so please let me know. I am going to use ShuffleNet to explain my question. 

In ShuffleNet, I have an input Node of dimension 5. This Node is transposed and outputs a node of shape [1,28,4,56,56]. This transposed Node, which we can define as OC2_DUMMY_2 is input into the next reshape layer. OC2_DUMMY_3, of int[4], is also input alongside the transposed Node, OC2_DUMMY_2. To my understanding, this means the Node of shape [1,28,4,56,56] is to be reshaped to a Node of 4 dimensions, gpu_0/shuffle_0_1. In the visualizer, this is what the above looks like:

```
Inputs:
    Data:   
        id: OC2_DUMMY_2
    Shape:
        id: OC2_DUMMY_3
        kind: Initializer
        type: int64[4]
        [
            1,
            112,
            56,
            56
        ]

Output:
    Reshaped:
        id: gpu_0/shuffle_0_1
```

How did the visualizer determine the output Node, when reshaped from a dimension of 5 to 4, will multiply the 2nd and 3rd dimensions (aka 28*4), to get the 112 for the 2nd dimension? Why didn't it do 4*56 or 56*56 to change the 3rd or 4th dimension? I know I am discussing one specific Reshape instance but I'm having the same confusion for all implementations of reshape in the visualizer. When I load the graph and get the .pb file, all the input DUMMY Nodes have for values is dim = 2 or dim = 4 in this case, and no mention of how the dimensions themselves will be resized. So where are the actual values of the dimensions themselves for the output Nodes being determined?

**Comment by lutzroeder** [comment]
The visualizer doesn't do this. It is done by the runtime library or framework you are using that implements `Reshape` when you run the model. The documentation for that framework should explain how `Reshape` works. For some frameworks like ONNX, TensorFlow or Keras you can click on the question mark next to the operator name to see some documentation.
